src/index.html

```diff
@@ -11,12 +11,112 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav-item">
+            <a
+              href="/apple"
+              class="nav_link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="/samsung"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="/smartphones"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="/laptops&computers"
```

Comment:
The `href` attribute value `/laptops&computers` might cause issues because the `&` character is not encoded. Consider using `/laptops%26computers` to ensure proper URL encoding.